HelpSource/Classes/Dictionary.schelp

```diff
@@ -284,13 +284,23 @@ d = (a: 5, b: 7, c: 1, d: 0);
 d.asSortedArray;
 ::
 
-method::asKeyValuePairs
-Return the values in an array of alternating key value pairs.
+method::asDict
+If no arguments are passed, return itself.  This is part of the link::Reference/Key-Value-Pairs:: interface.
+argument::mergeFunc
+This argument is not used, but exists to make the method compatible.
+argument::class
+A dictionary class to convert to, if given (conversion is done via code::newFrom::).
+
+method::asPairs
+Return the values in an array of alternating key value pairs.  This is part of the link::Reference/Key-Value-Pairs:: interface.
 code::
 d = (a: 5, b: 7, c: 1, d: 0);
-d.asKeyValuePairs;
+d.asPairs;
 ::
 
+method::asKeyValuePairs
+See asPairs.
```

Comment:
I would suggest adding a link here, I think, in case these methods ever get moved around and their definitions move apart.